packages/dune/dune.2.0.0/opam

```diff
@@ -26,7 +26,7 @@ homepage: "https://github.com/ocaml/dune"
 doc: "https://dune.readthedocs.io/"
 bug-reports: "https://github.com/ocaml/dune/issues"
 depends: [
-  "ocaml" {>= "4.06"}
+  ("ocaml" {>= "4.06"} | "ocaml" {< "4.06~~"} & "ocamlfind-secondary")
```

Comment:
```suggestion
  ("ocaml" {>= "4.06"} | ("ocaml" {< "4.06~~"} & "ocamlfind-secondary"))
```
Are you certain a parenthesis is not needed here? (I actually don't know)

Reply:
I think it might have been in opam 1, but not in 2. I tested this with a 4.08.1 and 4.02.3 switch and only the latter tried to install ocamlfind-secondary, so it seemed to be working as I expected! Feel free to add it for clarity, of course